luigi/worker.py

```diff
@@ -643,7 +649,9 @@ def _add(self, task, is_complete):
                        resources=task.process_resources(),
                        params=task.to_str_params(),
                        family=task.task_family,
-                       module=task.task_module)
+                       module=task.task_module,
+                       retry_policy_dict=_get_retry_policy_dict(task),
+                       deps_retry_policy_dicts=deps_retry_policy_dicts)
```

Comment:
As I've asked you in every iteration before. Why do we need the `deps_retry_policy_dicts`?